akka-docs/src/main/paradox/typed/actor-discovery.md

```diff
@@ -14,28 +14,38 @@ To use Akka Actor Typed, you must add the following dependency in your project:
   version=$akka.version$
 }
 
-## Introduction
+## Obtaining Actor references
 
-With @ref:[classic actors](../general/addressing.md) you would use `ActorSelection` to "lookup" actors. Given an actor path with
-address information you can get hold of an `ActorRef` to any actor. `ActorSelection` does not exist in Akka Typed, 
-so how do you get the actor references? You can send refs in messages but you need something to bootstrap the interaction.
+There are two general ways to obtain @ref:[Actor references](../general/addressing.md#what-is-an-actor-reference): by
+@ref:[creating](#creating-actors) actors and by discovery using the @ref:[Receptionist](#receptionist).
+Once an Actor is created others can get its reference by including the `ActorRef` in messages sent.
+Whereas with @ref:[classic actors](../general/addressing.md) you would use `ActorSelection` to "lookup" actors
+from an actor path with address information.
```

Comment:
Nicer to lead into these like this here.

Reply:
incidentally I updated this part earlier today in https://github.com/akka/akka/pull/27870
I think we can combine/merge the descriptions into something fantastic!
 ok if I merge mine first and you can rebase and rework the section here?